apps/backoffice-v2/src/common/hooks/usePagination/usePagination.tsx

```
@@ -25,18 +42,16 @@ export const usePagination = ({ totalPages }: { totalPages: number }) => {
 
   const onNextPage = useCallback(() => {
-    const pageNumber = Number(searchParams.page);
-    const nextPage = pageNumber + 1;
+    const nextPage = page + 1
 
     return defaultSerializer({
       ...searchParams,
       page: nextPage.toString(),
-    });
-  }, [searchParams]);
+    });oa
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)